### PR TITLE
Newsletter Settings: Wrap newsletter categories in its own settings section

### PR DESCRIPTION
**Pull request by mashikag**
Related to https://github.com/Automattic/wp-calypso/pull/82022

## Proposed Changes

* Wrap the newsletter categories in their own setting section: "Newsletter categories settings" 
* Replace the **Save settings** button from the bottom of the Newsletter categories settings, with the standard "Save settings" button found in settings section's header.
* Update the newsletter categories toggle to behave like the rest of toggles on the page, i.e. do not autosave on change, but instead require a user to "Save settings" after making a change

## Testing Instructions

* Go to `/settings/newsletter/$site-slug`
* Play around with the settings 

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

File: client/my-sites/site-settings/settings-podcast/main.tsx
Comment (mashikag) [inline client/my-sites/site-settings/settings-podcast/main.tsx:1]: @simison , I am removing this file. It must have been included in your PR as an accident. Left over after copy-paste I believe.

**Comment by yansern** [comment]
Marking this as **DO NOT MERGE** as this removes the autosaving toggle which is currently used to track how many users are enabling this feature over the weekends.

I can only follow up with the updated tracks event next week to support the changes made here.

**Comment by mashikag** [comment]
> Marking this as DO NOT MERGE as this removes the autosaving toggle which is currently used to track how many users are enabling this feature over the weekends.
> 
> I can only follow up with the updated tracks event next week to support the changes made here.

I addressed this with https://github.com/Automattic/wp-calypso/pull/82053/commits/2bbf3fcf3c4a484f4bc42cc3b55856993f7e7b69. Thanks for catching this out. 👍

**Review by yansern — APPROVED** [review]

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/9018489

Thank you @mashikag for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/9018489).

**Pull request merged**